src/printf.cpp

```diff
@@ -13,12 +13,19 @@
 // limitations under the License.
 
 #include <sstream>
+#include <stack>
 
 #include "printf.hpp"
 
 // Extract the conversion specifier from a format string
 char get_fmt_conversion(std::string_view fmt) {
     auto conversionSpecPos = fmt.find_first_of("diouxXfFeEgGaAcsp");
+    // Check if a valid conversion specifier was found
+    if (conversionSpecPos == std::string_view::npos) {
+        conversionSpecPos = -1;
```

Comment:
Why setting `conversionSpecPos` to `-1`?

Reply:
I was previously passing this as a ref to also get the position of the specifier, realized I don't need to, thanks for point this out;